src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1228,7 +1247,7 @@ int esbmc_parseoptionst::doit_termination()
   }
 
   log_status("Unable to prove or falsify the program, giving up.");
-  log_status("VERIFICATION UNKNOWN");
+  log_fail("VERIFICATION UNKNOWN");
```

Comment:
Why is an UNKNOWN verification result colored by `log_fail` and VERIFICATION FAILED below is also colored by `log_fail`? These two kinds of results have different meaning. (Applies to other places with UNKNOWN outputs, too.)